exercises/rotational-cipher/rotational-cipher.spec.js

```diff
@@ -0,0 +1,66 @@
+import RotationalCipher from './rotational-cipher';
+
+describe('Rotational cipher', () => {
+  test('rotate a by 0, same output as input', () => {
+    const expected = 'a';
+    const actual = RotationalCipher.rotate('a', 0);
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('rotate a by 1', () => {
+    const expected = 'b';
+    const actual = RotationalCipher.rotate('a', 1);
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('rotate a by 26, same output as input', () => {
+    const expected = 'a';
+    const actual = RotationalCipher.rotate('a', 26);
+
+    expect(actual).toEqual(expected);
+  });
+
```

Comment:
Looks like there is one test missing, belonging here, that is defined the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/rotational-cipher/canonical-data.json#L32):

```json
        {
          "description": "rotate m by 13",
          "property": "rotate",
          "text": "m",
          "shiftKey": 13,
          "expected": "z"
        },
```

Reply:
Correct! Must have missed that one, should be fixed now.